x-pack/plugins/observability/public/config/paths.ts

```diff
@@ -14,8 +14,10 @@ export const paths = {
     rules: RULES_PAGE_LINK,
     ruleDetails: (ruleId?: string | null) =>
       ruleId ? `${RULES_PAGE_LINK}/${encodeURI(ruleId)}` : RULES_PAGE_LINK,
-    alertDetails: (alertId?: string | null) =>
-      alertId ? `${ALERT_PAGE_LINK}/${encodeURI(alertId)}` : ALERT_PAGE_LINK,
+    alertDetails: (alertId?: string | null, ruleId?: string | null) =>
```

Comment:
@benakansara, `alertId` and `ruleId` should not be optional arguments. But the PR is  already merged

Reply:
@fkanout agreed, I think there is no use case where alertDetails() will be called without parameters. But in case it does, user will be redirected to Alerts page. It will not break the page. For code clarity, I agree to not make it optional. I will include this change in another PR.